test/test_bit_reproducibility.py

```diff
@@ -0,0 +1,122 @@
+"""Tests for model reproducibility"""
+
+import yaml
+import pytest
+from pathlib import Path
+
+from exp_test_helper import setup_exp
+
+
+class TestBitReproducibility():
+
+    @pytest.mark.checksum
+    def test_bit_repro_historical(self, output_path: Path, control_path: Path,
+                                  checksum_path: Path):
+        """
+        Test that a run reproduces historical checksums
+        """
+        # Setup checksum output directory
+        # NOTE: The checksum output file is used as part of `repro-ci` workflow
+        output_dir = output_path / 'checksum' 
+        output_dir.mkdir(parents=True, exist_ok=True)
+        checksum_output_file =  output_dir / 'CHECKSUM'
```

Comment:
In the `test_restart_repro` test, we name the checksum files stuff like `restart-1d-0-checksum.yaml` - should that be the convention, rather than `CHECKSUM`?

Reply:
I'll change it to `historical-3hr-checksum.yaml` as it runs part of the historical checksum tests, and using a model runtime of 3hrs to make it slightly more meaningful

Reply:
Changed the filename to be more descriptive in line with Jo's comment.